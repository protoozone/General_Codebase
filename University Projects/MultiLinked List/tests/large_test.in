NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
VIEW ALL
NEW 20
{0}
{1}
{2}
{3}
{4}
{5}
{6}
{7}
{8}
{9}
{10}
{11}
{12}
{13}
{14}
{15}
{16}
{17}
{18}
{19}
VIEW 20
VIEW-NESTED 20
VIEW ALL
DELETE 0 0
INSERT 0 0 big insert
VIEW-NESTED 20
EXIT