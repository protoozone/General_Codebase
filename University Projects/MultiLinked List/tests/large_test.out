List 0: 
List 1: 
List 2: 
List 3: 
List 4: 
List 5: 
List 6: 
List 7: 
List 8: 
List 9: 
List 10: 
List 11: 
List 12: 
List 13: 
List 14: 
List 15: 
List 16: 
List 17: 
List 18: 
List 19: 
Number of lists: 20
List 0
List 1
List 2
List 3
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
Nested 20: {List 0} -> {List 1} -> {List 2} -> {List 3} -> {List 4} -> {List 5} -> {List 6} -> {List 7} -> {List 8} -> {List 9} -> {List 10} -> {List 11} -> {List 12} -> {List 13} -> {List 14} -> {List 15} -> {List 16} -> {List 17} -> {List 18} -> {List 19}
{List 0} -> {List 1} -> {List 2} -> {List 3} -> {List 4} -> {List 5} -> {List 6} -> {List 7} -> {List 8} -> {List 9} -> {List 10} -> {List 11} -> {List 12} -> {List 13} -> {List 14} -> {List 15} -> {List 16} -> {List 17} -> {List 18} -> {List 19}
{} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {}
Number of lists: 21
List 0
List 1
List 2
List 3
List 4
List 5
List 6
List 7
List 8
List 9
List 10
List 11
List 12
List 13
List 14
List 15
List 16
List 17
List 18
List 19
Nested 20
INVALID COMMAND: DELETE
List 0: big insert
{big insert} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {} -> {}